sqlflow/pipeline.py

```
@@ -103,7 +102,7 @@ def consume_loop(self, max_msgs=None):
             logger.info(
                 'consumer loop ending: total messages / sec = {}'.format(self._stats.total_throughput_per_second),
             )
-            return self._stats
+        return self._stats
```

Didn't realize a `return` from a `finally` clause suppresses the error.